pkg/operator/ceph/file/dependent.go

```diff
@@ -35,6 +35,24 @@ func CephFilesystemDependents(clusterdCtx *clusterd.Context, clusterInfo *client
 
 	deps := dependents.NewDependentList()
 
+	// TODO: what if we are in external mode? how does external mode work w/ CSI?
+	// can PVCs from consumer cluster create subvolumes on provider, and do they appear in the "csi" subvolumegroup?
+	// if so, would this mean it's much better to list the PVs (PVCs)? and not do it by inspecting svgs/svs?
+	// the benefit to inspecting svgs/svs is that we implicitly block on CephNFS exports also
+	// how should we implement (if at all) allow deletion with volumes?
+	//
+	// svs = ceph fs subvolume ls <fsname>
+	// for sv in svs
+	//   dependents += sv.name
+	// svgs = ceph fs subvolumegroup ls <fsname>
+	// for svg in svgs:
+	//   svs = ceph fs subvolume ls <fsname> svg.name
+	//   for sv in svs
+	//     dependents += sv.name
```

Comment:
I started going through the pseudocode for blocking on cephfs dependents, and I need some info about corner cases, especially as relates to external mode. @leseb can we chat in huddle tomorrow?

Reply:
The current handling is to ignore empty subvolume groups in Ceph. We will still block on CephFilesystemSubVolumeGroup resources.